documentation/src/main/asciidoc/cheat-sheets/cache-configuration.adoc

```diff
@@ -0,0 +1,414 @@
++
+[source,xml]
+----
+<distributed-cache mode="ASYNC" />
+----
+
+== Cache configuration options
+
+.Encoding
+A format, identified by a media type, that {brandname} uses to store keys and values.
+
+TIP: {brandname} recommends the `application/x-protostream` media type. Protobuf gives you excellent performance as well as interoperability between client applications in different programming languages for both Hot Rod and REST endpoints.
+
+[source,xml]
+----
+<distributed-cache mode="SYNC">
+  <encoding>
+    <key media-type="application/x-protostream"/>
+    <value media-type="application/x-protostream"/>
+  </encoding>
+</distributed-cache>
+----
+
+.Expiration
+Expiration removes entries based on a lifespan or maximum idle time. Expiration takes effect across {brandname} clusters.
+
+Lifespan:: The maximum amount of time, in milliseconds, that cache entries can exist. If you set the value to -1, entries never expire.
+
+Max-idle:: The maximum amount of time, in milliseconds, that cache entries can remain idle. If no operations are performed on entries within the maximum idle time, the entries expire across the cluster. If you set the value to -1, entries never expire.
+
+TIP: Expiration with maximum idle time causes performance issues for `get()` update all instance, lifespan is better if very close.
+
+[source,xml]
+----
+<distributed-cache>
+  <expiration lifespan="5000"
+              max-idle="1000" />
+</distributed-cache>
+----
+
+.Statistics
+Start collecting statistics for your cache. Keep disabled for optimal performance.
+
+[source,xml]
+----
+<distributed-cache statistics="true" />
+----
+
+== {brandname} capabilities
+* Bounded
+* Indexed
+* Authorization
+* Backups
+* Transactional
+* Persistent
+
+== Bounded caches
+Control the size of the cache by configuring {brandname} to evict entries.
+
+.Eviction
+Eviction always takes place when you define either the max-size or the max-count (but not both) for the data container.
+
+Max size:: Defines the size of the data container in bytes. Eviction occurs when the approximate memory usage of the data container exceeds the maximum size.
++
+[source,xml]
+----
+<distributed-cache>
+  <memory max-size="12MB" when-full="REMOVE"/>
+</distributed-cache>
+----
+
+Max count:: Defines the size of the data container by number of entries. Eviction occurs after the container size exceeds the maximum count.
++
+[source,xml]
+----
+<distributed-cache>
+	<memory max-count="124" when-full="REMOVE"/>
+</distributed-cache>
+----
+
+.Eviction strategies
+* `REMOVE` {brandname} performs eviction. This is the default strategy when you define a size for the data container.
+IMPORTANT: Eviction removes entries from memory but not from persistent cache stores. To ensure that entries remain available after {brandname} evicts them, and to prevent inconsistencies with your data, you should configure persistent storage.
+
```

Comment:
The 'order' of eviction depends on the memory (off- on-heap) and it is not deterministic as caffeine is used as a performant implementation but it remove entries different to LRU, also it could remove different entries on different nodes which makes the cluster inconsistent!